content/experiments/qiskit-blocks.md

```diff
@@ -11,6 +11,7 @@ media:
   - https://content.minetest.net/uploads/Q5etnbmvSX.png
 
 source: https://github.com/JavaFXpert/QiskitBlocks
+belongsTo: experiments
```

Comment:
As before, this information is in the router. We should be able to extract it from there.

Reply:
The goal of `belongsTo` is to identify the page regardless of the URL changing over the time, so I think that is a good idea to keep this attribute